docker.go

```diff
@@ -98,6 +161,22 @@ func (container *Container) EmptyVolumes() *Container {
 	return container
 }
 
+// SetPersistentVolume defines persistent properties for volume
+//      size: size of the volume in MiB
+func (v *Volume) SetPersistentVolume(size int) *PersistentVolume {
```

Comment:
Is a persistent volume with a size of zero valid to Marathon? If yes, it's appropriate to have the `size` parameter. If not, we should drop it and leave it to the user to call `SetSize` or not.